### PR TITLE
192- Revise the model UI

### PR DESCRIPTION
**Pull request by shis20**
## Description

**Summary**: 

**Issue Link**: #192 

## Type of action

- [x] Bug fix 
- [x] New feature
- [ ] Requires a documentation update
- [ ] Others: 

## Checklist

- [x] My code style is consistent with the current project
- [x] I have done a brief review of my code
- [x] I have written readable comments
- [x] I have updated the documentation related to the changes I made
- [x] I have written reasonable tests and the updated version all passed
- [x] There should be no potential errors after the changes get merged into main

# Action Required

### CODE REVIEW

**Review by qinj15 — APPROVED** [review]
looks good!

File: src/app.py [hunk 1]
```diff
@@ -180,38 +202,53 @@ def add_course():
...
         upload_df_to_s3(df, s3, bucket_name, mock_data_file)
+        if current_page == "course_page":
+            return render_template(
+                "course_page.html",
+                username=username,
+                courses=courses,
+                current_page="course_page",
+            )
+        print(current_page)
+        print(courses)
```
Comment (katrina799) [inline src/app.py]: Could you please remove these print statements? Thanks

File: src/app.py [hunk 2]
```diff
@@ -116,6 +118,17 @@ def prority_predict():
     return render_template("model_page.html")
 
 
+# Router to smodel page
```
Comment (katrina799) [inline src/app.py]: A typo here `model` instead of `smodel`

**Review by katrina799 — COMMENTED** [review]
lgtm except few changes are required

**Review by songc120 — APPROVED** [review]

**Pull request merged**